src/screens/SendConfirmationSheet.tsx

```diff
@@ -289,6 +304,81 @@ export default function SendConfirmationSheet() {
         ]
   );
 
+  useEffect(() => {
+    if (isENS) {
+      const promises = [
+        estimateGasLimit({
+          address: accountAddress,
+          amount: 0,
+          asset: asset,
+          recipient: toAddress,
+        }),
```

Comment:
This function in the `SendSheet` sets the padded gas flag https://github.com/rainbow-me/rainbow/blob/develop/src/screens/SendSheet.js#L479

should we do it here?